File: java/program/TangkiBahanBakar.java
// TangkiBahanBakar.java
public class TangkiBahanBakar {
    // atribut
    private int kapasitas;
    private String jenisBahanBakar;

    // Constructor default
    public TangkiBahanBakar() {}

    // Constructor dengan parameter
    public TangkiBahanBakar(int kapasitas, String jenisBahanBakar) {
        this.kapasitas = kapasitas;
        this.jenisBahanBakar = jenisBahanBakar;
    }

    // Setter
    public void setKapasitas(int kapasitas) {
        this.kapasitas = kapasitas;
    }

    public void setJenisBahanBakar(String jenisBahanBakar) {
        this.jenisBahanBakar = jenisBahanBakar;
    }

    // Getter
    public int getKapasitas() {
        return kapasitas;
    }

    public String getJenisBahanBakar() {
        return jenisBahanBakar;
    }
}

File: python/program/tangki_bahan_bakar.py
# tangki_bahan_bakar.py
class TangkiBahanBakar:
    # constructor
    def __init__(self, kapasitas=0, jenis_bahan_bakar=""):
        self.kapasitas = kapasitas
        self.jenis_bahan_bakar = jenis_bahan_bakar

    # Setter
    def set_kapasitas(self, kapasitas):
        self.kapasitas = kapasitas

    def set_jenis_bahan_bakar(self, jenis_bahan_bakar):
        self.jenis_bahan_bakar = jenis_bahan_bakar

    # Getter
    def get_kapasitas(self):
        return self.kapasitas

    def get_jenis_bahan_bakar(self):
        return self.jenis_bahan_bakar
